Makefile

```diff
@@ -0,0 +1,31 @@
+BUNDLE_FLAGS = --build-arg BUNDLE_INSTALL_CMD='bundle install --jobs 20 --retry 5'
+
+ifdef DEPLOYMENT
+	BUNDLE_FLAGS = --build-arg BUNDLE_INSTALL_CMD='bundle install --without test'
+endif
+
+DOCKER_BUILD_CMD = docker-compose build $(BUNDLE_FLAGS)
+
+build:
+	$(MAKE) stop
+	$(DOCKER_BUILD_CMD)
+
+serve:
+	$(MAKE) build
+	docker-compose up -d
+
+lint:
+	$(MAKE) build
+	docker-compose run --rm app bundle exec govuk-lint-ruby
+
+test:
+	$(MAKE) serve
+	./mysql/bin/wait_for_mysql
+	docker-compose run --rm app rspec
+	$(MAKE) stop
```

Comment:
Is there a reason why we don't use a watch, e.g `guard` gem or similar, on those tests?

Reply:
Good idea, something to look into